lib/clean-url.js

```diff
@@ -1,6 +1,6 @@
 const { URL } = require('url')
 
-const replace = '***'
+const replace = '****'
```

Comment:
This makes tests fail, why the change?

Reply:
Oh, that was a mistake I did add an extra * while re-creating clean-url.js

Reply:
The issue seemed to be with the extra * and the mismatched tabs here, I used a 4 space tab while editing the code instead of a 2 spaces tab like the rest of the code, fixed now